### PR TITLE
update phd thesis

### PR DESCRIPTION
**Pull request by arun3688**

### CODE REVIEW

File: doc/bibliography/openmodelica.bib [hunk 1]
```diff
@@ -76,6 +76,17 @@ @BOOK{openmodelica.org:fritzson:2004
 %% [*2*] PhD/Lic theses
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@PHDTHESIS{openmodelica.org:lennart:phd:2016,
+  author = {Lennart Ochel},
+  title = {Petri-Netz-basierte Simulation biologischer Prozesse mit OpenModelica},
+  type = "{Doctoral thesis}",
+  year = {2016},
```
Comment (lochel) [inline doc/bibliography/openmodelica.bib]: It was published in 2017.

Comment (arun3688) [reply doc/bibliography/openmodelica.bib]: @lochel but in the pdf file, it says December 2016

File: doc/bibliography/openmodelica.bib [hunk 2]
```diff
@@ -76,6 +76,17 @@ @BOOK{openmodelica.org:fritzson:2004
...
+  address = {AG Bioinformatik Technische Fakultät},
+  school = {Universität Bielefeld},
+  doi = "https://pub.uni-bielefeld.de/download/2913956/2913957/diss_ochel_v1.1.pdf"
```
Comment (sjoelund) [inline doc/bibliography/openmodelica.bib]: That's not a DOI... It's a URL

File: doc/bibliography/openmodelica.bib (hunk 1 above)
Comment (sjoelund) [reply doc/bibliography/openmodelica.bib]: https://pub.uni-bielefeld.de/record/2913956 says 2017. Link to that

File: doc/bibliography/openmodelica.bib (hunk 2 above)
Comment (arun3688) [reply doc/bibliography/openmodelica.bib]: @sjoelund Updated

**Pull request merged**